daprdocs/content/en/developing-applications/debugging/bridge-to-kubernetes.md

```diff
@@ -0,0 +1,26 @@
+---
+type: docs
+title: "Bridge to Kubernetes support for Dapr services"
+linkTitle: "Bridge to Kubernets"
+weight: 300
+description: "Debug Dapr apps locally which still connected to your Kubernetes cluster"
+---
+
+Bridge to Kubernetes allows you to run and debug code on your development computer, while still connected to your Kubernetes cluster with the rest of your application or services. This type of debugging is often called *local tunnel debugging*. 
+
+{{< button text="Learn more about Bridge to Kubernetes" link="https://code.visualstudio.com/docs/containers/bridge-to-kubernetes" >}}
```

Comment:
Would you mind to use `https://aka.ms/bridge-vscode-dapr` instead? That way, we can adapt links to our documentation without needing to update yours.
Similar comment below for the isolation link: `https://aka.ms/bridge-isolation-vscode-dapr`

Reply:
fixed